server/app/assets/javascripts/main.ts

```diff
@@ -262,8 +262,15 @@ function attachStopPropogationListenerOnFormButtons() {
 /**
  * Disables default browser behavior where pressing Enter on any input in a form
  * triggers form submission. See https://github.com/civiform/civiform/issues/3872
+ * Except when the ignore element is present, the default form action would work
+ * See https://github.com/civiform/civiform/issues/5464
```

Comment:
After you seem my notes below please update this comment to include instructions on how to bypass the custom behavior that blocks hitting enter.